src/annotations.ts

```diff
@@ -0,0 +1,54 @@
+import { SQS } from 'aws-sdk'
+import { QueueConsumer } from './QueueConsumer'
+import { DeletionPolicy } from './DeletionPolicy'
+import { QueueConsumerConfig } from './QueueConsumerConfig'
+
+/**
+ * The @Listener method annotation to create a listener for AWS SQS messages.
```

Comment:
Mention the default interval `1000` if only a string is provided.

Reply:
Default interval clarified in 08845d8